gems/gibbon/CVE-2022-27311.yml

```diff
@@ -5,8 +5,7 @@ ghsa: vx9g-377x-xwxq
 url: https://github.com/amro/gibbon/pull/321
 title: Server side request forgery in gibbon
 date: 2022-04-26
-description: |
-  Gibbon v3.4.4 and below allows attackers to execute a Server-Side Request
-  Forgery (SSRF) via a crafted URL. A partial fix has been introduced in version 3.4.4,
-  however a complete fix has not yet been created. See Pull request 321 in github.com/amro/gibbon
-  for details.
+description: Gibbon v3.4.3 and below allows attackers to execute a Server-Side Request
+  Forgery (SSRF) via a crafted URL. This issue has been resolved in version 3.4.4
+patched_versions:
+- 3.4.4
```

Comment:
This should probably be `">= 3.4.4"` to indicate every version at and after 3.4.4 is considered patched.